README.md

```diff
@@ -48,12 +48,33 @@ import simulacrum as sm
 col_types = sm.ColTypes()
 col_types.add_coltype('ids', 'uuid')
 col_types.add_coltype('name', 'name')
-col_types.add_coltype('salaries', 'norm', mean=50000, sd=1000)
+col_types.add_coltype('salaries', 'norm', mean=50000, stdev=1000)
 col_types.add_coltype('ips', 'faker', provider='ipv6')
 
 data_set = sm.create(1000, coltypes=col_types.get_coltypes())
 ```
 
+For a quick demonstration, if you don't specify any types a dataframe is created with one of each of the available types:
+
+```python
```

Comment:
Yeah that's a good idea to build fast demo of the library or for educational purposes.